xcc/settings.py

```diff
@@ -87,15 +87,45 @@ class Config:  # pylint: disable=missing-class-docstring
         env_file = get_path_to_env_file()
         env_prefix = get_name_of_env_var()
 
+    def _sanity_check(self, key: str, val: str) -> None:
```

Comment:
I think we can make the function name more specific without sacrificing its generality:
```suggestion
    def _check_for_invalid_values(self, key: str, val: str) -> None:
```

Reply:
Did this in the code.